atlasexec/atlas_schema.go

```diff
@@ -804,8 +803,8 @@ func (c *Client) SchemaLint(ctx context.Context, params *SchemaLintParams) (*Sch
 }
 
 // SchemaStatsInspect runs the 'schema stats inspect' command.
-func (c *Client) SchemaStatsInspect(ctx context.Context, params *SchemaStatsInspectParams) ([]TableSizeMetric, error) {
-	args := []string{"schema", "stats", "inspect"}
+func (c *Client) SchemaStatsInspect(ctx context.Context, params *SchemaStatsInspectParams) (string, error) {
```

Comment:
```suggestion
func (c *Client) SchemaStatsInspect(ctx context.Context, params *SchemaStatsInspectParams) ([]byte, error) {
```

or `json.RawMessage`?

Reply:
Keeping consistency with other methods.